Dockerfile

```diff
@@ -1,7 +1,5 @@
 FROM harbor.intgdc.com/tools/gdc-java-8-jre:b057b53
 
-MAINTAINER LCM <lcm@gooddata.com>
```

Comment:
Why are you removing the maintainer ?

Reply:
Because it's deprecated https://docs.docker.com/engine/reference/builder/#maintainer-deprecated .

Reply:
ok.. good to know